### PR TITLE
Create chair-proposal-process.md

### PR DESCRIPTION
**Pull request by dshaw**
Documenting the process for our first exiting Chair, @dshaw.

### CODE REVIEW

**Review by justincormack — APPROVED** [review]

File: governance/chair-proposal-process.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# SIG-Security Chair Proposal Process
```
Comment (lumjjb) [inline governance/chair-proposal-process.md:1]: nit: Sentence case for header

**Review by lumjjb — APPROVED** [review]
LGTM. 1 style nit

File: governance/chair-proposal-process.md (hunk 1 above)
Comment (ultrasaurus) [inline governance/chair-proposal-process.md:1]:
```suggestion
# SIG-Security Chair proposal process
```

Comment (ultrasaurus) [reply governance/chair-proposal-process.md:1]: good catch @lumjjb -- adding suggested edit for @dshaw

File: governance/chair-proposal-process.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+1) SIG-Security Co-chairs operate as a team. The Co-Chair team seeks to maintain full coverage of the leadership capabilities across the key qualities of industry experience, hands-on cloud-native and security experience, as well as administrative experience needed to run a SIG-Security.
```
Comment (ultrasaurus) [inline governance/chair-proposal-process.md:3]: nit: noticed inconsistency in caps for "Co-chairs" and checked https://github.com/cncf/sig-security/blob/master/governance/roles.md#role-of-chairs -- seems we just use "Chair" in the role name.  I find that more readable

**Review by ultrasaurus — APPROVED** [review]
Approve.  Requested a small consistency fix, and also please link from https://github.com/cncf/sig-security/blob/master/governance/roles.md#role-of-chairs (other roles refer to selection process)

**Pull request merged**